### PR TITLE
Add python binding support

### PR DESCRIPTION
**Pull request by nplanel**
Implement python binding using ctypes.
Add a basic test unit : test_m68k.py

Signed-off-by: Nicolas PLANEL <nplanel@gmail.com>

### CODE REVIEW

File: include/capstone/m68k.h [hunk 1]
```diff
@@ -167,6 +168,8 @@ typedef enum m68k_fpu_size {
 
 // Type of size that is being used for the current instruction
 typedef enum m68k_size_type {
+	M68K_SIZE_TYPE_INVALID = 0,
```
Comment (emoon) [inline include/capstone/m68k.h]: This change will break printing of the instructions as it assumes it's set to _CPU by default. That needs to be added here then https://github.com/emoon/capstone/blob/next/arch/M68K/M68Kdasm.c#L3708

**Comment by emoon** [comment]
I have added these also

https://github.com/emoon/capstone/blob/next/include/capstone/capstone.h#L106

I wonder if we want to expose them in the python interface or that something that is added in the "general" python interface and not just for the M68K stuff?

**Comment by nplanel** [comment]
They will be exposed to python automatically by the bindings/const_generator.py script.

**Comment by emoon** [comment]
Merged the PR but like I wrote above that needs to be fixed

**Comment by nplanel** [comment]
Thanks, It's was already been fixed in the merge (set to CPU as default).
I will recheck/regenerate the m68k_const.py file

**Comment by emoon** [comment]
:+1:

**Pull request merged**